lib/LLVMIRCodeGen/CommandLine.h

```diff
@@ -20,5 +20,13 @@
 #include "llvm/Support/CommandLine.h"
 
 llvm::cl::OptionCategory &getLLVMBackendCat();
+/// Target to be used by the LLVMBackend.
+extern llvm::cl::opt<std::string> llvmTarget;
+/// Architecture to be used by the LLVMBackend.
+extern llvm::cl::opt<std::string> llvmArch;
+/// CPU to be used by the LLVMBackend.
+extern llvm::cl::opt<std::string> llvmCPU;
```

Comment:
can you put more comments here. I'm not sure I understand the difference between `arch` and `cpu`. What `cpu` exactly is?

Reply:
Done.